4.4.3/index.html

```diff
@@ -0,0 +1,49 @@
+
+<!doctype html>
+<html lang="en">
+  <head>
+      <title>Example 1</title>
+      <meta charset="utf-8" />
+      <meta http-equiv="x-ua-compatible" content="ie=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="style.css" />
+      <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.7.0/css/all.css'
+      integrity='sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ'
+      crossorigin='anonymous'>
+  </head>
+  <body>
+    <header>
+      <h1 class="header-text">Redes Sociales</h1>
+    </header>
+
+    <ul class="flex-container wrap">
```

Comment:
Maybe is better to use something else than an unordered list. There are two problems going around here:
1. The "ul" should be followed by "li" elements, not divs, as it breaks with semantic html
2. There is no list in the actual page layout, visually speaking. Therefore, there is no use of having one in the markup, there are alternative ways to create cards

Reply:
Thanks for the feedback, I changed it.

Please check it out again.

Reply:
Looks good, there's only one detail remaining, there's a duplicated icon for the twitter platform

![image](https://user-images.githubusercontent.com/7089148/62226298-3bf67b00-b377-11e9-8810-2af70d4b9118.png)

Reply:
Fixed